### ISSUE TITLE
Add ATD Model

### ISSUE THREAD

**Issue by NQevxvEtg** [unit8co/darts#1874]
The metrics seems ok. 

https://arxiv.org/pdf/2306.16950v1.pdf

Comparison with previous state-of-the-art methods on ETT test dataset.

Measurement metrics
Method MAE MSE

NLinear[33] 0.226 0.080
FiLM[37] 0.240 0.090
SCINet[18] 0.127 0.029
ATD (our) 0.058 0.006

**Comment by alexcolpitts96** [comment]
If the results this paper presents are correct, this may be the biggest jump in time series forecasting in years. That being said, I am very skeptical. They say it is a multi-modal model and then presented benchmark results on uni-modal datasets.

I am going to reach out to the authors and see if they have source code and more details they can provide about the model. It is not clear how they handled time series data since Figure 1 only indicates that it handles images and text. They may have fed the time series data in as an image and text (based on the figure), but that leads me to wonder how they suddenly crushed the benchmarks so significantly since their model should collapse to an LSTM and CNN feeding into an MLP.

**Comment by joshua-xia** [comment]
The metrics is really biggest improvement, can we reproduce the results on other datasets?

**Comment by alexcolpitts96** [comment]
I am still waiting on a reply from the authors. Their Github says they will release the source code after peer review.

Having dug more into the paper, I don't think it will actually belong in darts since it isn't a time series model. It is a multimodel (text + images) model and the published time series results were a part of their ablation testing.

**Comment by joshua-xia** [comment]
Yes, I also wonder where the text + images feature come from in the case of time series forecasting scenario.
